app/views/admin/users/index.html.haml

```diff
@@ -10,7 +10,7 @@
     %table.table.table-striped.table-bordered.table-hover.datatable
       %thead
         %th ID
-        %th State
+        %th Confirmation State
```

Comment:
![toggle](https://cloud.githubusercontent.com/assets/14155445/16849085/dbfdfd12-49c6-11e6-9392-0a8216e1f072.png)
Here is the actual problem, the side which has more number of characters gets squeezed everytime :stuck_out_tongue:. Can this be avoided?